app/controllers/api/v1/featured_tags/suggestions_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class Api::V1::FeaturedTags::SuggestionsController < Api::BaseController
+  before_action -> { doorkeeper_authorize! :read, :'read:featured_tags' }, only: [:index]
```

Comment:
Since adding new scopes for existing apps is a bit cumbersome, I wonder if maybe this should fall under `read:accounts` because it's part of an account "profile"

Reply:
Change to `read:accounts`